packit_service/worker/helpers/build/babysit.py

```diff
@@ -55,6 +57,13 @@
 logger = logging.getLogger(__name__)
 
 
+def celery_run_async(signatures: list[Signature]) -> None:
+    logger.debug("Signatures are going to be sent to Celery (from update_copr_build_state).")
```

Comment:
this can be called also from `update_testing_farm_run`, so the log should be adjusted